2024.html

```diff
@@ -95,6 +95,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://blakewatson.com/">Blake Watson</a></li>
             <li><a href="https://kayserifserif.place/">Katherine Yang</a></li>
             <li><a href="https://cssence.com/">Matthias Zöchling</a></li>
+            <li><a href="https://owly.fans">Cass &#xAB;Owly&#xBB; Python</a></li>
```

Comment:
Hello!

You can follow the instructions about sorting in the [README of this repository](https://github.com/css-naked-day/css-naked-day.github.io) (or the [comment above the list](https://github.com/css-naked-day/css-naked-day.github.io/pull/153/files#diff-24ac768f3b5003cd704e67986b8c414fa8320442ba2821cb55a2e9242d54c3e4L53-L59) :) ).

Thank you!